### PR TITLE
Feature/65 membership file export

### PR DESCRIPTION
**Pull request by EricTRL**
Upon saving, updating, or removing a member, certain information is stored in a CSV file for backup purposes.
The admin panel was made more usable (for the Member model and its related logs)

### CODE REVIEW

File: membership_file/admin.py [hunk 1]
```diff
@@ -1,11 +1,62 @@
 from django.contrib import admin
 from .models import Member, MemberLog, MemberLogField
-from django.contrib import admin
+from django.utils.html import format_html
+
+
+class HideRelatedNameAdmin(admin.ModelAdmin):
```
Comment (helaan) [inline membership_file/admin.py:6]: why this change instead of overriding the string representation?

File: membership_file/tests/tests_user.py [hunk 2]
```diff
@@ -57,6 +57,7 @@ def test_field_label(self):
 ################################################################
 
 # Tests the editing feature of a user's own membership info.
+@override_settings(MEMBERSHIP_FILE_EXPORT_PATH=None)
```
Comment (helaan) [inline membership_file/tests/tests_user.py:60]: why not test the export functionality instead?

**Review by helaan — APPROVED** [review]
Extra deployment steps required:
- merge settings.py (note: I should probably upstream these changes)
- run collectstatic as CSS was changed

File: membership_file/admin.py (hunk 1 above)
Comment (EricTRL) [reply membership_file/admin.py:6]: It'd make debugging a lot trickier; printing/loggin the instance would then show nothing.
A non-hacky solution would be to modify tha admin template, but that seemed overkill.

File: membership_file/tests/tests_user.py (hunk 2 above)
Comment (EricTRL) [reply membership_file/tests/tests_user.py:60]: That's done in a different (and more general) testcase in `tests_export.py`

**Comment by EricTRL** [comment]
Closes #65
